### PR TITLE
Feature/03: Create EN and FR translations

### PR DESCRIPTION
**Pull request by JrBour**
- Add translations on about and home page.
- Add translations on header and footer
- Set the default locale to `en`

### CODE REVIEW

File: templates/includes/_header.html.twig [hunk 1]
```diff
@@ -6,26 +6,26 @@
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
         <ul class="navbar-nav mr-auto">
             <li class="nav-item">
-                <a class="nav-link" href="#">Store</a>
+                <a class="nav-link" href="#">{{ 'header.nav.store'|trans }}</a>
```
Comment (iladrin) [inline templates/includes/_header.html.twig]: What about a "menu.store"?

File: templates/includes/_footer.html.twig [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+            <li><a href="#" title="#">{{ 'footer.nav.blog'|trans }}</a></li>
+            <li><a href="#" title="#">{{ 'footer.nav.register'|trans }}</a></li>
+            <li><a href="#" title="#">{{ 'footer.nav.login'|trans }}</a></li>
+            <li><a href="#" title="#">{{ 'footer.nav.french'|trans }}</a></li>
+            <li><a href="#" title="#">{{ 'footer.nav.english'|trans }}</a></li>
         </ul>
     </div>
     <div class="row">
-        <p><span class="text-muted">Designed and built with all the love of <b>ECVD school</b>.</span></p>
+        <p><span class="text-muted">{{ 'footer.disclaimer.start'|trans }}<b>{{ 'footer.disclaimer.end'|trans }}</b>.</span></p>
```
Comment (iladrin) [inline templates/includes/_footer.html.twig]: Please don't split the string into small pieces.

Comment (JrBour) [reply templates/includes/_footer.html.twig]: The translation can contains HTML tag ?

Comment (JrBour) [reply templates/includes/_footer.html.twig]: Done

**Pull request closed without merging**